### ISSUE TITLE
goto doesn't work for labels defined after the jump command

### ISSUE THREAD

**Issue by slechte** [MastersOfDesaster/Vongpiler#79]
The following code doesn't compile:

```
was ist das für 1 code?

gieb "Hallo" her?
g zu #goto du larry!!!

#goto
gieb "Welt" her?

1 nicer!!!
```

Jumps are not working when the label is defined after the actual jump. See the example above.

**Comment by benjamin-albrecht** [comment]
Closed with #81

**Comment by pkdevel** [comment]
talk mal normal ihr larries
